docs/installation/installation-using-docker-linux.md

```diff
@@ -28,33 +28,33 @@ cd project-base
 - *The `--no-install` option disables installation of the vendors - this will be done later in the Docker container.*
 - *The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
 
-### 2.1 Use install script
-In case you want to start demo of the app as fast as possible, you can now execute install script.
-
-```
-./scripts/install.sh
-```
-
-If you want to know more about what is happening during installation, continue with next step.
-
-### 2.2 Create docker-compose.yml file
+### 2. Installation
+Now, you have two options:
+- **Option 1:** In the case you want to start demo of the application as fast as possible, you can simply execute the installation script and that is all:
+    ```
+    ./scripts/install.sh
+    ```
+  After the script is finished, see [the last chapter of Application Setup Guide](./installation-using-docker-application-setup.md#2-see-it-in-your-browser) to get all the important information you might need right after the installation.  
```

Comment:
Do you this would make more sense? If so please update it on other places as well :slightly_smiling_face: 
```suggestion
  After the script is finished with installing the application, you can skip all other steps and see [the last chapter of Application Setup Guide](./installation-using-docker-application-setup.md#2-see-it-in-your-browser) to get all the important information you might need right after the installation.  
```

Reply:
yes :grin: